### PR TITLE
[OSDEV-975] Reporting. Number of facilities with at least one extended field.

### PR DESCRIPTION
**Pull request by Innavin369**
[OSDEV-975](https://opensupplyhub.atlassian.net/browse/OSDEV-975) Reporting. Number of facilities with at least one extended field.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -67,7 +67,7 @@ services:
     command:
       - "-b :8081"
       - "--reload"
-      - "--timeout=90"
+      - "--timeout=9000"
```
Comment (roman-stolar) [inline docker-compose.yml:70]: undo this changes pls

**Review by roman-stolar — DISMISSED** [review]
LGTM

File: docker-compose.yml [hunk 2]
```diff
@@ -128,7 +128,6 @@ services:
       retries: 5
     networks:
       - my-app
-
```
Comment (mazursasha1990) [inline docker-compose.yml:131]: Please, revert changes from `docker-compose.yml` as they are unrelated to your task.

**Review by roman-stolar — DISMISSED** [review]
LGTM

**Review by mazursasha1990 — APPROVED** [review]
LGTM

**Review by vladsha-dev — APPROVED** [review]
LGTM

File: docker-compose.yml (hunk 1 above)
Comment (Innavin369) [reply docker-compose.yml]: done
